README.md

```diff
@@ -33,35 +33,35 @@ providing necessary functionality for searching, discovering, data access/loadin
 
 `intake-esm` is a data cataloging utility built on top of [intake](https://github.com/intake/intake), [pandas](https://pandas.pydata.org/), and [xarray](https://xarray.pydata.org/en/stable/), and it's pretty awesome!
 
-- Opening an ESM collection definition file: An ESM (Earth System Model) collection file is a JSON file that conforms
-  to the [ESM Collection Specification](https://github.com/NCAR/esm-collection-spec). When provided a link/path to an esm collection file, `intake-esm` establishes
+- Opening an ESM catalog definition file: An ESM (Earth System Model) catalog file is a JSON file that conforms
+  to the [ESM Collection Specification](https://github.com/NCAR/esm-collection-spec). When provided a link/path to an esm catalog file, `intake-esm` establishes
   a link to a database (CSV file) that contains data assets locations and associated metadata
-  (i.e., which experiment, model, the come from). The collection JSON file can be stored on a local filesystem
+  (i.e., which experiment, model, the come from). The catalog JSON file can be stored on a local filesystem
   or can be hosted on a remote server.
 
   ```python
 
   In [1]: import intake
 
-  In [2]: col_url = "https://gist.githubusercontent.com/andersy005/7f416e57acd8319b20fc2b88d129d2b8/raw/987b4b336d1a8a4f9abec95c23eed3bd7c63c80e/pangeo-gcp-subset.json"
+  In [2]: cat_url = "https://gist.githubusercontent.com/andersy005/7f416e57acd8319b20fc2b88d129d2b8/raw/987b4b336d1a8a4f9abec95c23eed3bd7c63c80e/pangeo-gcp-subset.json"
 
```

Comment:
Let's change this to use the new sample/tutorial catalogs

Reply:
Done